zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/ElasticsearchStorage.java

```diff
@@ -299,7 +301,10 @@ CheckResult ensureIndexTemplatesAndClusterReady(String index) {
     try {
       version(); // ensure the version is available (even if we already cached it)
       ensureIndexTemplates(); // called only once, so we have to double-check health
-      AggregatedHttpRequest request = AggregatedHttpRequest.of(GET, "/_cluster/health/" + index);
+      final RequestHeaders requestHeaders = RequestHeaders.builder(GET, "/_cluster/health/" + index)
```

Comment:
maybe add a comment on why `endOfStream(true)`?

Reply:
Not sure this is solved with this change. Also, I think I have to fix the code in Armeria. 😅 
Let's rebuild a couple of times more.

Reply:
travis will run out of breath :)

Reply:
Oops, I got this error.
https://travis-ci.org/github/openzipkin/zipkin/builds/735943469#L7319

Reply:
Let me revisit this later after I finished my training session. 😅

Reply:
this comment is out-of date, so closing